README.md

```diff
@@ -43,11 +17,13 @@ There are two ways to install:
      gpio_pin: D0
      
      # (Optional) Path to the leapmmw_sensor.h file relative to your esphome configuration directory.
-     header_file: leapmmw_sensor.h
+     # header_file: leapmmw_sensor.h
```

Comment:
Are we sure we want this commented out as the default?  I am not sure people understand the correlation between this line and being able to make changes to sensor settings.  Might be good to expand on that.

Reply:
Happy to clarify the comment but in terms of functionality it is no change. The file path is included in the substitutions in the package. This just allows one to override the location in their config (if they haven’t saved it in the top level esphome directory).

Reply:
ah, that makes sense.  Still getting a handle on this new fancy packaging :)